miden-proto/src/account.rs

```diff
@@ -0,0 +1,19 @@
+/// Represents every type of account in the rollup:
```

Comment:
I added the generated code to the repo. This is much better to use with a language server (jump to definion and what not works without extra configuration)